### DISCUSSION TITLE
Getting ical address of a Google Calendar shared with "See only free/busy" permission

### DISCUSSION THREAD

**Discussion by mderazon** [derekantrican/GAS-ICS-Sync#410] [Q&A]
Is it possible for the owner of the calendar who shared the calendar with `See only free/busy` option to provide an ical address that has the limited visibility ?
The only options I see are either
1. Share the public ical url in which case it only works if the calendar is public
2. Share the secret ical url in which case the full calendar details will be shown

Basically, I want to copy events from that calendar to my calendar, but I want them to shown without event details. The only option I see to get the ical url for that calendar is to make it public to the world

**Comment by jonas0b1011001** [comment]
You can sync the secret url and exclude any unwanted details by removing them from https://github.com/derekantrican/GAS-ICS-Sync/blob/90157f5c82a8f4bd0930f24b82a8e909215bccc4/Helpers.gs#L350
